aliyun_driver.go

```diff
@@ -778,13 +791,50 @@ func (d *Driver) PostCheck(ctx context.Context, info *types.ClusterInfo) (*types
 	return info, nil
 }
 
+func getClientset(info *types.ClusterInfo) (*kubernetes.Clientset, error) {
```

Comment:
This func shares common logic with `PostCheck`, please refactor that (I would probably have `PostCheck` call this func).